### ISSUE TITLE
Benchmarking perf(FLOP/s) and peak_perf(FLOPS/s)

### ISSUE THREAD

**Issue by dasswap** [peterahrens/ReproBLAS#3]
Testing with ReproBLAS with the existing benchmarks is yielding very offputting answers when looking at perf(FLOPs/s) and peak_perf(FLOPS/s) measures. Here is a snapshot of one of the results obtained from some testing while trying to determine the arithmetic intensity of the operations we're working with.

**N = 2048, perf(FLOPS/S) of ReproBLAS operations**
![flops-s_2048](https://user-images.githubusercontent.com/29139049/31839508-3fef9cd6-b596-11e7-8d4e-b48d080479da.JPG)

**N = 2048, peak_perf(FLOPS/S) of ReproBLAS operations**
![peakflops-s_2048](https://user-images.githubusercontent.com/29139049/31839509-4011ea8e-b596-11e7-967c-d900148b7748.JPG)

Both images work with N = 2048 elements, but the difference in FLOPS/s and peak FLOPS/s is too stark to not be worth mentioning - a multiplicative factor of 10^5. Just wanted to clarify what exactly peak_perf(FLOPS/s) is measuring, and why this difference is so high.

**Comment by willow-ahrens** [comment]
Okay. I really really need to fix #2. 
In the meantime, lets debug this.
peak perf is supposed to tell you the theoretical peak performance. It is a constant which does not depend on the observed runtime.
perf is just supposed to tell you the performance.
Here is the code which spits out the numbers:
```
    if self.attribute == "time(s)":
      return output["time"] / output["trials"]
    elif self.attribute == "peak_time(s)":
      return terminal.get_peak_time(output)
    elif self.attribute == "perf(FLOP/s)":
      return (output["trials"] * terminal.get_flop_count(output)) / max(output["time"], sys.float_info.min)
    elif self.attribute == "peak_perf(FLOP/s)":
      return (output["trials"] * terminal.get_flop_count(output)) / max(terminal.get_peak_time(output), sys.float_info.min)
    elif self.attribute == "norm(Hz)":
      return (output["trials"] * output["normalizer"]) / max(output["time"], sys.float_info.min)
    elif self.attribute == "peak_norm(Hz)":
      return (output["trials"] * output["normalizer"]) / max(terminal.get_peak_time(output), sys.float_info.min)
    elif self.attribute == "freq(Hz)":
      return (output["trials"] * output["input"]) / max(output["time"], sys.float_info.min)
    elif self.attribute == "peak_freq(Hz)":
      return (output["trials"] * output["input"]) / max(terminal.get_peak_time(output), sys.float_info.min)
    elif self.attribute == "peak(%)":
      return (100.0 * terminal.get_peak_time(output) * output["trials"])/max(output["time"], sys.float_info.min)
```
could you put a printf in there and spit out `terminal.get_peak_time(output)` and `output["time"]`?

**Comment by willow-ahrens** [comment]
Also, I suspect this is a peak perf problem because if we were getting 10^15 flops per second on your laptop that would be pretty groundbreaking...
